site/helm-integration.md

```diff
@@ -230,6 +231,24 @@ setting `helmOperator.configureRepositories.enable` to `true` for the
 flux Helm release, or as shown in the commented-out sections of the
 [example deployment](../deploy-helm/helm-operator-deployment.yaml).
 
+#### Azure ACR repositories
+
+For Azure ACR repositories, the entry in `repositories.yaml` created by
+running `az acr helm repo add` is unsufficient for the Helm operator.
+Instead you will need to [create a service principal](https://docs.microsoft.com/en-us/azure/container-registry/container-registry-auth-service-principal#create-a-service-principal)
+and use the plain text id and password this gives you. For example:
+
+```yaml
+- caFile: ""
```

Comment:
I would add the stable repo too:
```yaml
- caFile: ""
  cache: stable-index.yaml
  certFile: ""
  keyFile: ""
  name: stable
  password: ""
  url: https://kubernetes-charts.storage.googleapis.com
```

Reply:
Doesn't that distract from the actual information we are trying to get to the user?

Reply:
If they add only the ACR repo, any stable chart will fail. Maybe you could add a note on this and keep the yaml shot.

Reply:
I think this note is indeed applicable to all users with a custom `repositories.yaml`, will add it to the top section.